src/main/java/seedu/address/logic/commands/AddProjectCommand.java

```diff
@@ -55,12 +55,7 @@ public CommandResult execute(Model model) throws CommandException {
             editEmployeeDescriptor.setProject(toAdd);
             new EditCommand(targetIndex, editEmployeeDescriptor).execute(model);
 
-
-            //removes employee from previous project
-            if (employeeToAdd.getProject().name != "") {
-                //TODO: get project by name
-            }
-            toAdd.addEmployee(employeeToAdd);
+            toAdd.addEmployee(lastShownList.get(targetIndex.getZeroBased()));
```

Comment:
Is writing `lastShownList.get(targetIndex.getZeroBased())` different from using `employeeToAdd`? Perhaps I overlooked something but if they are the same, maybe you can just use `employeeToAdd` since it's already there.

Reply:
They are in fact the same, thanks for catching that!